geniuspaste/src/geniuspaste.c

```diff
@@ -94,6 +94,54 @@ PLUGIN_SET_TRANSLATABLE_INFO(LOCALEDIR, GETTEXT_PACKAGE, PLUGIN_NAME,
                              PLUGIN_VERSION, "Enrico Trotta <enrico.trt@gmail.com>")
 
 
+/* Windows compatibility for resolving system plugin data path.
+ * This only works if the plugin's data is installed in the same tree as the
+ * plugin DLL, which might not be true (as the DLL has to be in Geany's libdir
+ * but the plugin's data might be anywhere in theory) */
```

Comment:
Wait.  I think I just had an epiphany:  if we use the code I have here, but on a `libgeany-plugins` library, which would be installed in GP's own tree, it would be correct no matter where the Geany and GP's tree are far apart.

But maybe it's just being too nice, as maybe indeed we rely on both being next to one another.